thinkhazard_processing.yaml

```diff
@@ -1,37 +1,78 @@
 sqlalchemy.url: postgresql://www-data:www-data@localhost:5432/thinkhazard_processing
+data_path: /var/sig
 
 hazard_types:
   FL:
     hazard_type: river_flood
-    global:
-      return_periods:
-        LOW: 1000
-        MED: 50
-        HIG: 25
-    local:
-      return_periods:
-        LOW: 100
-        MED: 50
-        HIG: 25
+    preprocessed: False
```

Comment:
Since most hazard types are not preprocessed, this line should not be required IMO.